Packs/FraudWatch/Integrations/FraudWatch/FraudWatch.py

```diff
@@ -0,0 +1,857 @@
+        maybe_unaware_date)
+    return aware_time_date.strftime('%Y-%m-%d')
+
+
+def get_and_validate_positive_int_argument(args: Dict, argument_name: str) -> Optional[int]:
+    """
+    Extracts int argument from Demisto arguments.
+    If argument exists, validates that:
+    - min <= argument's value.
+
+    Args:
+        args (Dict): Demisto arguments.
+        argument_name (str): The name of the argument to extract.
+
+    Returns:
+        - If argument exists and is equal or higher than min, returns argument.
+        - If argument exists and is lower than min, raises DemistoException.
+    """
+    argument_value = arg_to_number(args.get(argument_name), arg_name=argument_name)
+
+    if not argument_value:
+        return None
+
+    if not MINIMUM_POSITIVE_VALUE <= argument_value:
+        raise DemistoException(f'{argument_name} should be equal or higher than {MINIMUM_POSITIVE_VALUE}')
+
+    return argument_value
+
+
+def get_time_parameter(arg: Optional[str]):
+    """
+    Returns date time object with aware time zone if 'arg' exists.
+    If no time zone is given, sets timezone to UTC.
+    Args:
+        arg (str): The argument to turn into aware date time.
+
+    Returns:
+        - If 'arg' is None, returns None.
+        - If 'arg' is exists returns date time.
+    """
+    maybe_unaware_date = arg_to_datetime(arg, is_utc=True)
+    if not maybe_unaware_date:
+        return None
+
+    aware_time_date = maybe_unaware_date if maybe_unaware_date.tzinfo else UTC_TIMEZONE.localize(
+        maybe_unaware_date)
+    return aware_time_date
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def fetch_incidents_command(client: Client, params: Dict):
+    """
+    Retrieves new incidents from FraudWatch.
+    Uses Demisto last run parameters in the following way:
+    - 'last_fetch_day' - The latest day fetching was done. Used in 'from_date' to the API call
+       to fetch only incidents from the relevant day and forward, as any date before 'last_fetch_day'
+       have already been fetched or skipped intentionally and there is no need to fetch incidents
+       before 'last_fetch_day'.
+    - 'last_fetch_date_time' - The latest time fetching was done. Because FraudWatch API 'from_date' is accurate
+       per day, and fetching incidents usually happens more than once a day, this is needed to make sure that
+       same incident is not fetched more than once.
+
+    Args:
+        client (Client): FraudWatch client to perform API call to fetch incidents.
+        params (Dict): Demisto params.
+
+    Returns:
+        A tuple of Incidents that have been fetched, and the new run parameters.
+    """
+    last_run = demisto.getLastRun()
+    brand = params.get('brand')
+    status = params.get('status')
+    limit = params.get('max_fetch')
+
+    fetch_time_string = params.get('first_fetch', '5 days').strip()
+    first_fetch_time = get_time_parameter(fetch_time_string)
```

Comment:
because i use 2 formats for this date, in lines
    first_fetch_time = get_time_parameter(fetch_time_string)
    from_date = last_run.get('last_fetch_day', first_fetch_time.strftime(FRAUD_WATCH_DATE_FORMAT))
    last_fetch_time = arg_to_datetime(last_run.get('last_fetch_date_time', first_fetch_time.isoformat()))
    @ShahafBenYakir